hazelcast/src/main/resources/hazelcast-default.yaml

```diff
@@ -23,32 +23,55 @@
 # To learn how to configure Hazelcast, please see the Reference Manual
 # at https://hazelcast.org/documentation/
```

Comment:
I think this docs link is stale

Reply:
Could be. I don't know if it will be, so asked on slack - https://hazelcast.slack.com/archives/C026A3BMTQD/p1626081125146000?thread_ts=1626079394.145700&cid=C026A3BMTQD

Reply:
Got my answer. It's going to be https://docs.hazelcast.com/. Updated all occurrences.